src/runtime/virtcontainers/pkg/oci/utils.go

```diff
@@ -772,6 +773,12 @@ func addHypervisorVirtioFsOverrides(ocispec specs.Spec, sbConfig *vc.SandboxConf
 		sbConfig.HypervisorConfig.VirtioFSDaemon = value
 	}
 
+	if value, ok := ocispec.Annotations[vcAnnotations.VirtioFSExtraArgs]; ok {
+		var parsedValue []string
+		_ = json.Unmarshal([]byte(value), &parsedValue)
```

Comment:
@pmores, why are you silently ignoring an error here?
What about checking for the error and returning in case we fail to unmarshal the parsedValue?

Reply:
@fidencio It depends on what we're trying to achieve - I can do that if that's the desired handling.  I'm sorry but admittedly I don't have much deeper context about semantics of this code so I might misjudge.

Reply:
@pmores, thanks this is exactly what I was looking for.